CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# PureScript Community Code of Conduct
+
+The PureScript community follows some simple ground rules to protect safe, inclusive spaces to discuss the PureScript programming language.
+
+All community members are expected to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## What's our purpose?
+
+The PureScript community channels are places to discuss the PureScript programming language. 
+
+* We talk about the language, its related libraries and infrastructure, and our experiences using it.
+* We help PureScript programmers learn the language, and we ask for help becoming stronger programmers ourselves. 
```

Comment:
I think the comma before the "and" should be removed. Perhaps this is a writing style thing but all the other commas have some grammatical purpose while this one appears to indicate a pause.

Reply:
The comma here could be removed or kept as-is. Again, I don't have an opinion.